doc/content/examples/divertor_monoblock/index.md

```diff
@@ -0,0 +1,257 @@
+The 2D monoblock consisted of three materials:
+
+- W monoblock (r(mm)>8.5),
+
+- Cu interlayer (7.5 < r(mm)<8.5),
+
+- CuCrZr tube (6.0 < r(mm)<7.5), and
+
+- H2O cooling (r(mm)<6.0),
+
+where $r=\sqrt{x^2+y^2}$ and the center point (x,y) = (0,0) is defined at the center of the CuCrZr tube.
+
+!media examples/figures/divertor_monoblock_mesh.png
+  id=fig:mesh
+  caption=2D monoblock: (left) geometry and mesh; (right) temperature distribution. This corresponds to Fig. 1 in Ref. [!cite](Shimada2024114438).
+  style=display:block;margin-left:auto;margin-right:auto;width:50%
+
+MOOSE is equipped with a set of user-friendly built-in mesh generators for creating meshes based on simple geometries (e.g., a monoblock).
+The built-in mesh generator `ConcentricCircleMeshGenerator` is used to create meshes for the 2D monoblock geometry.
+Only the left-hand (−14.0 < $x$ (mm)<0 & −14.0 < $y$ (mm)<14.0) portion of the monoblock is used,
+such that the mesh size is halved by assuming symmetry at the $x = 0$ plane.
+This example uses a total of 9,418 nodes, creating 28,402 degrees of freedom in non-linear systems.
+The finer mesh size and time step are required to compute the trapping-limited diffusion regime, as described in TMAP8 verification case [ver-1d](ver-1d.md).
+However, a set of relatively low detrapping energies (< 0.85 eV) is used here to reduce the computing time via the relatively coarse mesh size.
+
+In the input file, the mesh is defined as:
+
+!listing test/tests/divertor_monoblock/divertor_monoblock.i link=false block=Mesh
+
+
+### Variables and governing equations
+
+To simulate tritium and thermal transport, we define two sets of PDEs.
+First, the strong form of the mass conservation equation for solute (mobile) T atoms, $C_s$, is written as:
+
+\begin{equation} \label{eq:concentration}
+\frac{\partial C_s}{\partial t} + \nabla \cdot (-D \nabla C_s) + \alpha_t \frac{C_t^e}{N} C_s - \alpha_r C_t = 0.
+\end{equation}
+
+We use three sets of mass conservation equations to calculate the behaviors of solute T atoms in three different materials (i.e., W, Cu, CuCrZr).
+Second, the strong form of the conservation of energy equation is written as:
+
+\begin{equation} \label{eq:temperature}
+\rho c_p \frac{\partial T}{\partial t} - \nabla \cdot (k_T \nabla T) = 0.
+\end{equation}
+
+[tab:variables] lists all the symbols and definitions used in [eq:concentration] and [eq:temperature].
+
+!table id=tab:variables caption=Nomenclature of all the variables and physical parameters used in [eq:concentration] and [eq:temperature].
+| Symbol | Variable or Physical Property | Unit |
+| --- | --- | --- |
+| $C_s$ | Concentration of solute (mobile) species | m$^{−3}$ |
+| $C_t$ | Concentration of trapped species | m$^{−3}$ |
+| $T$ | Temperature | K |
+| $D$ | Diffusivity of solute (mobile) species | m$^{2}$ /s |
+| $K_s$ | Solubility of solute (mobile) species | m$^{−3}$/Pa$^{1/2}$ |
```

Comment:
```suggestion
| $K_s$ | Solubility of solute (mobile) species | m$^{−3}$ Pa$^{-1/2}$ |
```

Style consistency. Also, this physical parameter is not used in either of the equations mentioned in the caption and should be reorganized to be closer to where it is used. This is not the only one I see ($\alpha_{r0}$, $E_{dt}$) that mis-align with the caption.

Alternatively a section for "nomenclature" could be created near the top of the problem description so that it serves as a key for the equations and relationships in the document for those physically relevant quantities with units (as opposed to the purely FEM nomenclature).